app/models/movie.rb

```diff
@@ -1,3 +1,13 @@
 class Movie < ApplicationRecord
   belongs_to :studio
+  has_many :movie_actors
+  has_many :actors, through: :movie_actors
+
+  def actors_list
+    actors.order(age: :asc)
+  end
```

Comment:
Great job with this, just note that ascending is the default, so `actors.order(:age)` is the same thing. :)